infrastructure/2024-10-28/main.tf

```diff
@@ -0,0 +1,18 @@
+module "concepts_pipeline" {
+  source = "../stack"
+
+  namespace = local.pipeline_date
+
+  # This is the namespace of the works catalogue pipeline
+  # that this concept pipeline should subscribe to for changes.
+  catalogue_namespace = "2024-08-15" // This is automatically bumped by the catalogue-api repo
```

Comment:
There is a script in the catalogue-api repository bumping this date. We'll need to make minor changes to the script to work with this refactoring.

Reply:
See https://github.com/wellcomecollection/catalogue-api/pull/826